dicom-viewer.yml

```diff
@@ -0,0 +1,15 @@
+---
+services:
+  ocis:
+    depends_on:
+      dicomviewer-init:
+        condition: service_completed_successfully
+
+  dicomviewer-init:
+    image: owncloud/web-app-dicom-viewer/dicom-viewer-1.0.0
```

Comment:
we still have to create the corresponding docker image!

Reply:
@S-Panta this is the path to the docker image, ensure that it is correct after creating the actual docker image on  docker hub. If we follow the same structure/naming conventions like other ownCloud apps seem to use, it should be https://hub.docker.com/u/owncloud/web-app-dicom-viewer and for the tag dicom-viewer-1.0.0

Reply:
The name should be `owncloud/web-app-dicom-viewer:latest` for now
https://hub.docker.com/r/owncloud/web-app-dicom-viewer/tags